supplierProduct/migrations/0001_initial.py

```diff
@@ -0,0 +1,68 @@
+            fields=[
+                ('user_name', models.CharField(help_text='User Name',
+                                               max_length=32,
+                                               primary_key=True,
+                                               serialize=False,
+                                               validators=[django.core.validators.MinLengthValidator(6)])),
+                ('first_name', models.CharField(help_text='First Name',
+                                                max_length=32,
+                                                validators=[django.core.validators.MinLengthValidator(1)])),
+                ('last_name', models.CharField(help_text='Last Name',
+                                               max_length=32,
+                                               validators=[django.core.validators.MinLengthValidator(1)])),
+                ('password', models.CharField(help_text='Password',
+                                              max_length=16,
+                                              validators=[django.core.validators.MinLengthValidator(6)])),
+                ('user_type', models.CharField(choices=[('client', 'client'), ('supplier', 'supplier')],
+                                               help_text='User Type - supplier/client',
+                                               max_length=16,
+                                               validators=[django.core.validators.MinLengthValidator(6)])),
+                ('business_name', models.CharField(help_text='Business Name',
+                                                   max_length=32,
+                                                   validators=[django.core.validators.MinLengthValidator(6)])),
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+        migrations.CreateModel(
+            name='supplierProduct',
+            fields=[
+                ('suppler_product_id',
+                 models.IntegerField(primary_key=True,
+                                     serialize=False)),
+                ('price',
+                 models.IntegerField(
+                                     validators=[django.core.validators.MinLengthValidator(1)])),
+                ('quantity',
+                 models.IntegerField(
+                                     validators=[django.core.validators.MinLengthValidator(1)])),
+                ('qr_code',
+                 models.ForeignKey(
+                                    on_delete=django.db.models.deletion.CASCADE,
+                                    to='product.product')),
+                ('user_name',
+                 models.ForeignKey(
+                                    on_delete=django.db.models.deletion.CASCADE,
+                                    to='supplierProduct.supplier')),
+            ],
+            options={
+                'unique_together': {('suppler_product_id', 'qr_code')},
```

Comment:
`suppler_product_id` is the primary key, meaning it will always be unique. 
Maybe you were referring to `user_name` instead.